api/app/views/mno_enterprise/jpi/v1/admin/teams/_team.json.jbuilder

```diff
@@ -0,0 +1,20 @@
+@all_apps ||= MnoEnterprise::App.all.to_a
```

Comment:
Do we really to load all apps for each team here?
Couldn't we load and include team -> app_instances -> app
Especially if it's just to display the logo